mission-8/src/controllers/notifications.controller.ts

```diff
@@ -0,0 +1,59 @@
+import { Request, Response, NextFunction } from 'express';
+import { NotificationsService } from '../services/notifications.service.js';
+
+export class NotificationsController {
+  notificationsService = new NotificationsService();
```

Comment:
```suggestion
  notificationsService = NotificationsService.,getInstance();
```

생성자를 private하게 만들어 둔 싱글톤 클래스라 컴파일/런타임 에러 둘 다 발생하지 않을까요?